bindings/pyroot/cppyy/cppyy-backend/clingwrapper/src/clingwrapper.cxx

```diff
@@ -811,6 +811,7 @@ T CallT(Cppyy::TCppMethod_t method, Cppyy::TCppObject_t self, size_t nargs, void
     T t{};
     if (WrapperCall(method, nargs, args, (void*)self, &t))
         return t;
+    throw std::runtime_error("failed to resolve function");
```

Comment:
Is this change in behavior documented (i.e., `throw`ing instead of `return`ing $-1$)?  (same applies for line 845 and some others below)